skel/.mozilla/firefox/jzlvae43.default/prefs.js

```diff
@@ -1,104 +0,0 @@
-user_pref("browser.history_expire_days.mirror", 180);
-user_pref("browser.migration.version", 6);
-user_pref("browser.places.createdSmartBookmarks", false);
-user_pref("browser.places.importDefaults", false);
-user_pref("browser.places.leftPaneFolderId", -1);
-user_pref("browser.places.migratePostDataAnnotations", false);
-user_pref("browser.places.smartBookmarksVersion", 4);
-user_pref("browser.places.updateRecentTagsUri", false);
-user_pref("browser.rights.3.shown", true);
-user_pref("browser.search.selectedEngine", "Google");
-user_pref("browser.search.suggest.enabled", false);
-user_pref("browser.search.update", false);
-user_pref("browser.startup.homepage_override.buildID", "20120713224749");
-user_pref("browser.startup.homepage_override.mstone", "14.0.1");
-user_pref("browser.tabs.loadInBackground", true);
-user_pref("browser.tabs.loadBookmarksInBackground", true);
-user_pref("datareporting.healthreport.uploadEnabled", false);
-user_pref("distribution.canonical.bookmarksProcessed", true);
-user_pref("extensions.adblockplus.checkedadblockinstalled", true);
-user_pref("extensions.adblockplus.checkedtoolbar", true);
-user_pref("extensions.adblockplus.frameobjects", false);
-user_pref("extensions.adblockplus.showintoolbar", false);
-user_pref("extensions.adblockplus.showsubscriptions", false);
-user_pref("extensions.blocklist.pingCountTotal", 2);
-user_pref("extensions.blocklist.pingCountVersion", -1);
-user_pref("extensions.bootstrappedAddons", "{}");
-user_pref("extensions.databaseSchema", 12);
-user_pref("extensions.disabledObsolete", true);
-user_pref("extensions.enabledAddons", "globalmenu@ubuntu.com:3.2.5,langpack-en-ZA@firefox.mozilla.org:14.0.1,langpack-en-GB@firefox.mozilla.org:14.0.1,ubufox@ubuntu.com:2.0.3,{972ce4c6-7e08-4474-a285-3208198ce6fd}:14.0.1");
-user_pref("extensions.enabledItems", "ubufox@ubuntu.com:0.9rc2,langpack-en-GB@firefox-3.6.ubuntu.com:3.6,{972ce4c6-7e08-4474-a285-3208198ce6fd}:3.6.10");
-user_pref("extensions.installCache", "[{\"name\":\"app-system-share\",\"addons\":{\"ubufox@ubuntu.com\":{\"descriptor\":\"/usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/ubufox@ubuntu.com\",\"mtime\":1341671421000}}},{\"name\":\"app-global\",\"addons\":{\"{972ce4c6-7e08-4474-a285-3208198ce6fd}\":{\"descriptor\":\"/usr/lib/firefox/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}\",\"mtime\":1342550500000},\"langpack-en-GB@firefox.mozilla.org\":{\"descriptor\":\"/usr/lib/firefox/extensions/langpack-en-GB@firefox.mozilla.org.xpi\",\"mtime\":1342227406000},\"langpack-en-ZA@firefox.mozilla.org\":{\"descriptor\":\"/usr/lib/firefox/extensions/langpack-en-ZA@firefox.mozilla.org.xpi\",\"mtime\":1342227406000},\"globalmenu@ubuntu.com\":{\"descriptor\":\"/usr/lib/firefox/extensions/globalmenu@ubuntu.com\",\"mtime\":1342550480000}}}]");
-user_pref("extensions.pendingOperations", false);
-user_pref("extensions.update.enabled", false);
-user_pref("extensions.update.notifyUser", false);
```

Comment:
You might need this pref in mozilla.cfg, otherwise the user gets prompted for extension updates